Inheritance/src/character/extendsCharacter/extendsPawPatrol/Marshall.java

```diff
@@ -0,0 +1,23 @@
+package character.extendsCharacter.extendsPawPatrol;
+
+import character.extendsCharacter.PawPatrol;
+
+public class Marshall extends PawPatrol {
+    public Marshall(String name, String function, String superPower, String breed) {
+        super(name, function, superPower, breed);
+    }
+
+    public Marshall() {
+        super("character.extendsCharacter.extendsPawPatrol.Marshall",
```

Comment:
character.extendsCharacter.extendsPawPatrol.Marshall - should be only Marshall

Reply:
Done